lib/redux/actions.dart

```diff
@@ -0,0 +1,4 @@
+class ChangeScreenMode {
```

Comment:
Why you have redux files in 
/lib/redux
and in 
/lib/screens/redux?

They are completely identical

Reply:
Actually not sure now who copypasted from whom